src/app.py

```diff
@@ -160,48 +169,33 @@ def get_record_count(obj_config):
             return db.get_table_size('swap_positions')
 
 
-def get_args():
-    """ Configure a parser to retrieve & parse command-line arguments
-    input by the user.
-
-    Returns
-    -------
-    namespace
-        Namespace is populated with observed arguments, unless none observed,
-        in which case default values are returned. Access elements via '.'
-        convention, i.e. parser.parse_args().config
-    """
-    # Configure expected command line args & default values thereof
-    parser = ArgumentParser(description='''Random financial data
-                                        generation. For more information,
-                                        see the README''')
-    parser.add_argument('--config', default='src/config.json',
-                        help='JSON Configuration File Location')
-    return parser.parse_args()
-
-
 def get_fb_config(file_builders, file_extension):
```

Comment:
can we name this get_file_builder_config. Use fully descriptive names if there is no character limit in the programming langauge